midp/gfx.js

```diff
@@ -896,74 +896,31 @@ var currentlyFocusedTextEditor;
     }
 
     function setClip(g, x, y, width, height) {
```

Comment:
This function doesn't ever set `g.clipped` to false so if a `Graphics` object ever becomes clipped, we will always think it is clipped